kotlin-html-generator/src/main/kotlin/dev/scottpierce/html/generate/model/GeneratedElement.kt

```diff
@@ -236,6 +241,11 @@ sealed class GeneratedElement(
                 callingContext = Context.Body,
                 childrenContext = Context.Body
             ),
+            Normal(
+                tagName = "summary",
+                callingContext = Context.Body,
+                childrenContext = Context.Body
+            ),
```

Comment:
@Tmpod for future reference - all I did was add these elements into this file, and run the generator